docs/azure-monitor-logs.md

```diff
@@ -0,0 +1,57 @@
+# Looking at test logs and metrics in Azure Monitor
+
+1. Open the `thief-ai` resource in the Azure portal
+2. On the left pane, click "Logs" in the "Monitoring" section
```

Comment:
Did you know in markdown you can number every bullet `1.` and it will automatically increment when rendered?

Reply:
Yes, I did know that, but that feels like "more than formatting" to me, so I generally don't use it.

Reply:
ah, I personally like it because adding or editing steps doesn't create a tiresome task of updating all the numbers, or forgetting and have them not make sense.